### PR TITLE
BTFS-2410: BTFS password reset issue

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: core/commands/wallet.go [hunk 1]
```diff
@@ -250,7 +250,9 @@ var walletPasswordCmd = &cmds.Command{
 	Arguments: []cmds.Argument{
 		cmds.StringArg("password", true, false, "password of BTFS wallet."),
 	},
-	Options: []cmds.Option{},
+	Options: []cmds.Option{
+		cmds.StringOption(passwordOptionName, "p", "password"),
```
Comment (taiyangc) [inline core/commands/wallet.go:254]: why another password? argument password = original password?

Comment (laipogo) [reply core/commands/wallet.go:254]: old password

Comment (laipogo) [reply core/commands/wallet.go:254]: arg.password is new password to encrypt the priv_key.
options.password is old password to decrypt the cipher_priv_key and validate.

Comment (taiyangc) [reply core/commands/wallet.go:254]: Need to update helptext to reflect this. Not easy to understand.

**Review by taiyangc — APPROVED** [review]

**Pull request merged**